examples/hybrid/advection_agnesi.jl

```diff
@@ -0,0 +1,376 @@
+using Test
+using StaticArrays, LinearAlgebra
+
+import ClimaCore:
+    ClimaCore,
+    slab,
+    Spaces,
+    Domains,
+    Meshes,
+    Geometry,
+    Topologies,
+    Spaces,
+    Fields,
+    Hypsography,
+    Operators
+using ClimaCore.Geometry
+
+using Logging: global_logger
+using TerminalLoggers: TerminalLogger
+global_logger(TerminalLogger())
+
+const MSLP = 1e5 # mean sea level pressure
+const grav = 9.8 # gravitational constant
+const R_d = 287.058 # R dry (gas constant / mol mass dry air)
+const γ = 1.4 # heat capacity ratio
+const C_p = R_d * γ / (γ - 1) # heat capacity at constant pressure
+const C_v = R_d / (γ - 1) # heat capacity at constant volume
+const R_m = R_d # moist R, assumed to be dry
+
+function warp_agnesi(coord; h₀ = 10, a = 10_000)
+    x = Geometry.component(coord, 1)
+    return h₀ * a^2 / (x^2 + a^2)
+end
+
+# set up function space
+function hvspace_2D(
+    xlim = (-π, π),
+    zlim = (0, 4π),
+    helem = 30,
+    velem = 80,
+    npoly = 4;
+    stretch = Meshes.Uniform(),
+    warp_fn = warp_agnesi_peak,
+)
+    # build vertical mesh information with stretching in [0, H]
+    FT = Float64
+    vertdomain = Domains.IntervalDomain(
+        Geometry.ZPoint{FT}(zlim[1]),
+        Geometry.ZPoint{FT}(zlim[2]);
+        boundary_tags = (:bottom, :top),
+    )
+    vertmesh = Meshes.IntervalMesh(vertdomain, stretch, nelems = velem)
+    vert_face_space = Spaces.FaceFiniteDifferenceSpace(vertmesh)
+    # build horizontal mesh information
+    horzdomain = Domains.IntervalDomain(
+        Geometry.XPoint{FT}(xlim[1]),
+        Geometry.XPoint{FT}(xlim[2]);
+        periodic = true,
+    )
+    # Construct Horizontal Mesh + Space
+    horzmesh = Meshes.IntervalMesh(horzdomain; nelems = helem)
+    horztopology = Topologies.IntervalTopology(horzmesh)
+    quad = Spaces.Quadratures.GLL{npoly + 1}()
+    hspace = Spaces.SpectralElementSpace1D(horztopology, quad)
+    # Apply warp
+    z_surface = warp_fn.(Fields.coordinate_field(hspace))
+    f_space = Spaces.ExtrudedFiniteDifferenceSpace(
```

Comment:
I hard reset onto this branch and attempted to run this example driver resulting in the following method error :
```
2D Extruded Terrain Warped Space: Error During Test at /Users/asridhar/Research/Codes/ClimaCore.jl/test/Spaces/terrain_warp.jl:62
  Got exception outside of a @test
  MethodError: no method matching ClimaCore.Spaces.ExtrudedFiniteDifferenceSpace(::ClimaCore.Spaces.CellFace, ::ClimaCore.Spaces.SpectralElementSpace1D{ClimaCore.Topologies.IntervalTopology{ClimaCore.Meshes.IntervalMesh{ClimaCore.Domains.IntervalDomain{ClimaCore.Geometry.XPoint{Float64}, Nothing}, LinRange{ClimaCore.Geometry.XPoint{Float64}, Int64}}, NamedTuple{(), Tuple{}}}, ClimaCore.Spaces.Quadratures.GLL{5}, ClimaCore.Geometry.CartesianGlobalGeometry, ClimaCore.DataLayouts.IFH{ClimaCore.Geometry.LocalGeometry{(1,), ClimaCore.Geometry.XPoint{Float64}, Float64, StaticArrays.SMatrix{1, 1, Float64, 1}}, 5, Array{Float64, 3}}, ClimaCore.DataLayouts.IFH{Float64, 5, Array{Float64, 3}}}, ::ClimaCore.Hypsography.TerrainWarpedIntervalTopology{ClimaCore.Topologies.IntervalTopology{ClimaCore.Meshes.IntervalMesh{ClimaCore.Domains.IntervalDomain{ClimaCore.Geometry.ZPoint{Float64}, Tuple{Symbol, Symbol}}, LinRange{ClimaCore.Geometry.ZPoint{Float64}, Int64}}, NamedTuple{(:bottom, :top), Tuple{Int64, Int64}}}}, ::ClimaCore.Geometry.CartesianGlobalGeometry, ::ClimaCore.DataLayouts.VIFH{ClimaCore.Geometry.LocalGeometry{(1, 3), ClimaCore.Geometry.XZPoint{Float64}, Float64, StaticArrays.SMatrix{2, 2, Float64, 4}}, 5, Array{Float64, 4}}, ::ClimaCore.DataLayouts.VIFH{ClimaCore.Geometry.LocalGeometry{(1, 3), ClimaCore.Geometry.XZPoint{Float64}, Float64, StaticArrays.SMatrix{2, 2, Float64, 4}}, 5, Array{Float64, 4}})
  Closest candidates are:
    ClimaCore.Spaces.ExtrudedFiniteDifferenceSpace(::S, ::H, ::T, ::GG, ::LG, ::LG, ::LGG, ::LGG) where {S<:ClimaCore.Spaces.Staggering, H<:ClimaCore.Spaces.AbstractSpace, T<:ClimaCore.Topologies.AbstractIntervalTopology, GG<:ClimaCore.Geometry.AbstractGlobalGeometry, LG, LGG} at ~/Research/Codes/ClimaCore.jl/src/Spaces/hybrid.jl:13
```
Is this simply pointing to my environment / Manifest being out of sync?

Reply:
We checked in the Manifest, so you should be able to just `Pkg.instantiate()` to get the same result.

Reply:
This was a rebase error, it should be corrected now.